common/utils/src/main/resources/error/error-classes.json

```
@@ -2423,6 +2423,12 @@
     ],
     "sqlState" : "42883"
   },
+  "RULE_ID_NOT_FOUND" : {
+    "message" : [
+      "Rule id not found for <ruleName>. Please modify RuleIdCollection.scala if you are adding a new rule."
```

Let's highlight the rule name in the error message as we do for other parameters, and use the default quoting:
```suggestion
      "Not found an id for the rule name \"<ruleName>\". Please modify RuleIdCollection.scala if you are adding a new rule."
```